packages/lib/modules/portfolio/PortfolioTable/PortfolioTable.tsx

```diff
@@ -202,7 +202,9 @@ export function PortfolioTable() {
           </Card>
         ) : (
           <Center border="1px dashed" borderColor="border.base" h="400px" rounded="lg">
-            <ConnectWallet size="lg" variant="primary" />
+            <div>
```

Comment:
maybe you didn't notice it but we're using chakra ui components in the layout where possible